go/cmd/vtgate/vtgate.go

```diff
@@ -36,12 +37,10 @@ import (
 	"vitess.io/vitess/go/vt/vterrors"
 	"vitess.io/vitess/go/vt/vtgate"
 	"vitess.io/vitess/go/vt/vtgate/planbuilder/plancontext"
-
-	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 )
 
 var (
-	cell                           = "test_nj"
+	cell                           = ""
```

Comment:
I couldn't find how to mark a flag as required in pflags. We already enforce the flag's requirement manually by failing if the value is empty.

Reply:
`fs.MarkRequired`

Reply:
I checked, there is nothing like `fs.MarkRequired` in `pflags`.

Reply:
ohhh shoot i forgot; it's a [cobra thing](https://pkg.go.dev/github.com/spf13/cobra#MarkFlagRequired)

Reply:
I tried adding it, but it doesn't really work. I can still call vtgate without the cell flag specified and flag parsing doesn't complain.